brevproduksjon/src/main/java/no/nav/foreldrepenger/fpformidling/integrasjon/dokgen/dto/avslagfp/ForeldrepengerAvslagDokumentdata.java

```diff
@@ -79,13 +83,13 @@ public boolean equals(Object object) {
             that.mottattDato) && Objects.equals(gjelderFødsel, that.gjelderFødsel) && Objects.equals(barnErFødt, that.barnErFødt) && Objects.equals(
             annenForelderHarRett, that.annenForelderHarRett) && Objects.equals(antallBarn, that.antallBarn) && Objects.equals(halvG, that.halvG)
             && Objects.equals(klagefristUker, that.klagefristUker) && Objects.equals(lovhjemmelForAvslag, that.lovhjemmelForAvslag) && Objects.equals(
-            kreverSammenhengendeUttak, that.kreverSammenhengendeUttak) && Objects.equals(avslåttePerioder, that.avslåttePerioder);
+            kreverSammenhengendeUttak, that.kreverSammenhengendeUttak) && Objects.equals(gjelderMor, that.gjelderMor) && Objects.equals(avslåttePerioder, that.avslåttePerioder);
```

Comment:
Gammel kode, men er avslåttePerioder sortert på fom? Ellers bør man erstatte med 
avslåttePerioder.size() == that.avslåttePerioder.size() and avslåttePerioder.containsAll(that.avslåttePerioder)
Mange List.equals sjekker at element for element er like.

Reply:
ja, de er sortert på fom